packages/design-system/src/components/SearchBar/SearchBar.md

```diff
@@ -0,0 +1,46 @@
+### SearchBar
+
+
+### Compact search bar
+
+Used for mobile devices or smaller resolution designs
+
+```js
+const onChange = value => {
+  console.log('Search term: ', value);
+}
+<SearchBar compact onChange={onChange} />
+```
+
+### Search bar with onChange handler (see console)
+
+Most common use case for search bar is with onChange handler. It can be enhanced with debounce method to reduce many callbacks
+
+```js
+const onChange = value => {
+  console.log('Search term: ', value);
+}
+<SearchBar onChange={onChange} />
+```
+
+### Search bar with onSubmit handler (see console)
```

Comment:
no need to add this "(see console)", dev is able to see the source code of the example